esmvalcore/preprocessor/_regrid.py

```diff
@@ -104,9 +104,51 @@ def parse_cell_spec(spec):
     return dlon, dlat
 
 
-def _stock_cube(spec, lat_offset=True, lon_offset=True):
+def _stock_cube(latdata, londata, circular: bool = False):
```

Comment:
probably better to rename to eg `_regional_stock_cube` and below to `_global_stock_cube`

Reply:
Thanks for pointing this out. The function names were a bit of a poor choice from my side. I have updated them to make more sense 👍

Reply:
all in good time, we can't both code and be Jane Austens at the same time :laughing: